Sources/Interpreter/Interpreter.swift

```diff
@@ -296,6 +299,54 @@ public struct Interpreter {
     topOfStack.registers[i] as? Address
   }
 
+  /// Returns the value of `x` if it has a address, or `nil` if it does not.
+  func address(_ x: Operand) -> Address? {
```

Comment:
```suggestion
  func asAddress(_ x: Operand) -> Address? {
```

I guess given the semantics, this should read like a cast.